### PR TITLE
Feature/userstory 5

### PR DESCRIPTION
**Pull request by AstridOttmann**

### CODE REVIEW

File: components/MainNavigation/StyledNavigation.js [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+      `;
+    } else if (variant === "language") {
+      return css`
+        display: flex;
+        justify-content: flex-start;
+        gap: 1em;
+        background: whitesmoke;
+      `;
+    }
+  }}
```
Comment (F-Kirchhoff) [inline components/MainNavigation/StyledNavigation.js:27]: This pattern is ok for 2 variants, but a general approach is writing a function for each variant with the `&&` operator:
```js
${({variant}) => variant === 'main' && css`
   ... css here
`}
${({variant}) => variant === 'language' && css`
   ... css here
`}

File: pages/index.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default function HomePage() {
           {isFound === false && (
             <>
               <p>added new word</p>
-              <Link href="/words">show entry {``}</Link>
+              <Link href="/words">show entry {`>>`}</Link>
```
Comment (F-Kirchhoff) [inline pages/index.js]: instead of this you might consider using an icon or unix character here.

File: pages/words/index.js [hunk 3]
```diff
@@ -3,22 +3,54 @@ import globalTranslations from "@/public/store";
...
 export default function WordsPage() {
   const [translationList, setTranslationList] = useAtom(globalTranslations);
+  const [selectedLanguage, setSelectedLanguage] = useState("");
+  //
+  const usedLanguagesWithDublicates = translationList.map((language) => {
+    return language.language;
+  });
+  //removing duplicates
+  const usedLanguages = [...new Set(usedLanguagesWithDublicates)]; // https://dev.to/soyleninjs/3-ways-to-remove-duplicates-in-an-array-in-javascript-259o
+  console.log("usedLanguages", usedLanguages);
```
Comment (F-Kirchhoff) [inline pages/words/index.js]: please remove logs 🕵️

File: pages/words/index.js [hunk 4]
```diff
@@ -3,22 +3,54 @@ import globalTranslations from "@/public/store";
...
 
 export default function WordsPage() {
   const [translationList, setTranslationList] = useAtom(globalTranslations);
+  const [selectedLanguage, setSelectedLanguage] = useState("");
+  //
+  const usedLanguagesWithDublicates = translationList.map((language) => {
+    return language.language;
+  });
+  //removing duplicates
+  const usedLanguages = [...new Set(usedLanguagesWithDublicates)]; // https://dev.to/soyleninjs/3-ways-to-remove-duplicates-in-an-array-in-javascript-259o
```
Comment (F-Kirchhoff) [inline pages/words/index.js:19]: nice!

File: pages/words/index.js [hunk 5]
```diff
@@ -3,22 +3,54 @@ import globalTranslations from "@/public/store";
...
   }
 
+  // displays the list with the entries in the selected language
+  const filteredTranslations = translationList.filter((translation) => {
+    if (selectedLanguage) {
+      return translation.language === selectedLanguage;
+    }
+    return true;
+  });
+  console.log("filtered", filteredTranslations);
```
Comment (F-Kirchhoff) [inline pages/words/index.js]: 🕵️

File: pages/words/index.js [hunk 6]
```diff
@@ -3,22 +3,54 @@ import globalTranslations from "@/public/store";
...
+
   return (
     <>
```
Comment (F-Kirchhoff) [inline pages/words/index.js]: this fragment is not needed here.

File: pages/words/index.js [hunk 7]
```diff
@@ -3,22 +3,54 @@ import globalTranslations from "@/public/store";
...
+          selectedLanguage={selectedLanguage}
+          onLanguageSelection={handleLanguageSelection}
+        />
         <h1>My words</h1>
         <StyledList>
-          {translationList.map((translation) => (
+          {filteredTranslations.map((translation) => (
             <ListEntry key={translation.id}>
               <p>{translation.word}</p>
               <small>({translation.language})</small>
```
Comment (F-Kirchhoff) [inline pages/words/index.js]: for your SVGIcon you need an aria-label for folks who use screenreader.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
I only found small things, great work here, especially the usage of Set for filtering the array.

File: components/MainNavigation/StyledNavigation.js (hunk 1 above)
Comment (AstridOttmann) [reply components/MainNavigation/StyledNavigation.js:27]: Thank you, I didn't know.

File: pages/index.js (hunk 2 above)
Comment (AstridOttmann) [reply pages/index.js]: Good idea, I changed it.

File: pages/words/index.js (hunk 3 above)
Comment (AstridOttmann) [reply pages/words/index.js]: Done!

File: pages/words/index.js (hunk 6 above)
Comment (AstridOttmann) [reply pages/words/index.js]: Removed it!

File: pages/words/index.js (hunk 7 above)
Comment (AstridOttmann) [reply pages/words/index.js]: I add aria-label in SVGIcon component. Correct?

**Review by RobinWitt — APPROVED** [review]
looks good to me!

**Pull request merged**